client/lib/post-types-list/list.js

```diff
@@ -73,7 +75,7 @@ PostTypesList.prototype.fetch = function( siteId ) {
  * @return {array} services
  **/
 PostTypesList.prototype.parse = function( data ) {
-	return data.post_types;
+	return values( omit( data, '_headers' ) );
```

Comment:
> This is not coming from the API response directly. With the HTTP envelope enabled, our v1 API returns body, code, headers properties, and WP-API returns body, headers, status properties.

We're adding `_headers` in both request handlers:

`wpcom-proxy-request`: https://github.com/Automattic/wpcom-proxy-request/blob/master/index.js#L397

`wpcom-xhr-request`: https://github.com/Automattic/wpcom-xhr-request/blob/master/index.js#L101